ajax-upgradetab.php

```diff
@@ -44,6 +44,7 @@
         ob_clean();
     }
     echo '{wrong token}';
+    http_response_code(401);
```

Comment:
since php 5.4.
maybe prefer?
```
header('HTTP/1.1 401 Unauthorized');
```

Reply:
The module is compatible with PHP 5.6 minimum, so we're fine.

However, I have no idea which solution is better.

Reply:
It looks they basically have the same result. HTTP 401 will be properly set by PHP: https://github.com/php/php-src/blob/fc42ac2d78e28134bc46ef58c289586a0907dd79/main/http_status_codes.h#L46

Reply:
Oh my bad, thinking it was 5.2+ because of 1.6 compliance ^^